financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/FinancialConnectionsSheetNativeViewModel.kt

```diff
@@ -379,6 +367,28 @@ internal data class FinancialConnectionsSheetNativeState(
     )
 }
 
+sealed class WebAuthFlowState : Parcelable {
+    @Parcelize
+    object Uninitialized : WebAuthFlowState()
+
+    @Parcelize
+    object InProgress : WebAuthFlowState()
+
+    @Parcelize
+    data class Success(
+        val url: String
+    ) : WebAuthFlowState()
+
+    @Parcelize
+    object Canceled : WebAuthFlowState()
+
+    @Parcelize
+    data class Failed(
+        val message: String,
+        val reason: String?
+    ) : WebAuthFlowState()
+}
```

Comment:
Created a Parcelable state that can be saved and restored on process kills. 

That way we can recover the web state pre-process kill and act accordingly.